cellar_rebalancer/src/cellars/uniswapv3.rs

```diff
@@ -158,11 +166,44 @@ impl<T: 'static + Middleware> UniswapV3CellarState<T> {
     }
 }
 
-// Struct for UniswapV3CellarTickInfo
-#[derive(Clone, Debug)]
-pub struct UniswapV3CellarTickInfo {
-    pub(crate) token_id: U256,
-    pub(crate) tick_upper: i32,
-    pub(crate) tick_lower: i32,
-    pub(crate) weight: u32,
+pub struct UniswapV3CellarAllocator;
+
+#[async_trait]
```

Comment:
This is the only import from `axum` used in the project. Is it necessary? I believe `tonic` also reexports this.

Reply:
The directive is necessary but you are correct, we can remove the `axum` dep.